src/main/java/de/uplanet/lucy/connectorapi/examples/google/drive/datagroup/GoogleDriveDataGroupService.java

```diff
@@ -419,4 +410,27 @@ private String _getStringFromResponse(final HttpResponse p_response) throws IOEx
 		String l_string = writer.toString();
 		return l_string;
 	}
+
+	private String _extreactStringFieldValue(IConnectorRecord p_record, String p_fieldName)
+	{
+		IConnectorField l_field;
+		try
+		{
+			l_field = p_record.getFieldByName(p_fieldName);
+		}
+		catch (Exception l_e)
+		{
+			return null;
+		}
+
+		if (l_field == null)
+			return null;
+
+		Object l_value = l_field.getValue().getValue();
+
+		if (l_value instanceof String)
```

Comment:
Please use `ValueHolderHelper.getStringFromVH(l_field.getValue())` instead of using instanceof.